bin/bazel-temp

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```

Comment:
Is this supposed to have `-temp` in its name?

Reply:
I named it `-temp` since I'm planning to write a `bin/bazel` that will be more general than this script today